tools/src/Tools_Docgen.resi

```diff
@@ -5,11 +5,16 @@ type field = {
   optional: bool,
   deprecated?: string,
 }
+
+@tag("kind")
+type constructorPayload = | @as("inlineRecord") InlineRecord({fields: array<field>})
+
 type constructor = {
   name: string,
   docstrings: array<string>,
   signature: string,
   deprecated?: string,
+  constructorPayload?: constructorPayload,
```

Comment:
@zth This field name (`constructorPayload`) is not matching the field name given in [`stringifyDetail in DocExtraction.ml`](https://github.com/rescript-lang/rescript-vscode/blob/719f8a29f0691d7e7ad3029ae4d64acb6459b4ed/analysis/src/DocExtraction.ml#L121) (`items`).
If I'm not mistaken, both names need to be identical: either `items` or `constructorPayload`.

Reply:
@zth changed `items` from array to object then better rename field to `payload` or `params`?